### PR TITLE
Features/docu 533

### PR DESCRIPTION
**Pull request by benradstone**

### CODE REVIEW

File: docs/quickstarts/installation/s3-adlsg2_bi_lm.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+sidebar_label: S3 & ADLS Gen2 with LiveMigrator
+---
+
+Use this quickstart if you want to configure Fusion to replicate between an S3 bucket and an ADLS Gen2 container in either direction using WANdisco LiveMigrator.
+
+What this guide will cover:
+
+- Installing WANdisco Fusion using the [docker-compose](https://docs.docker.com/compose/) tool.
+- Integrating WANdisco Fusion with S3 and ADLS Gen2 storage.
+- Performing a sample data migration in each direction.
```
Comment (catrionabarker) [inline docs/quickstarts/installation/s3-adlsg2_bi_lm.md:13]: Maybe
Performing sample data migrations in both directions.

File: docs/quickstarts/installation/s3-adlsg2_bi_lm.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+1. Click on the **Settings** cog for the **adls2** storage, and fill in the details for your ADLS Gen2 container. See the [Info you will require](#info-you-will-require) section for reference.
+
+1. Check the **Use Secure Protocol** box.
+
+1. Click **Apply Configuration** and wait for this to complete.
+
+## Migration
+
+Follow the steps below to demonstrate migration of HCFS data to and from the S3 bucket and the ADLS Gen2 container.
```
Comment (catrionabarker) [inline docs/quickstarts/installation/s3-adlsg2_bi_lm.md:108]: Maybe
... HCFS data between the S3 ...

File: docs/quickstarts/installation/s3-adlsg2_bi_lm.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+
+   * S3 bucket = `/s3_to_adls2`
+     * See the S3 docs ([Amazon console](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/create-folder.html) / [AWS CLI](https://docs.aws.amazon.com/cli/latest/reference/workdocs/create-folder.html)) for guidance.
+   * ADLS Gen2 container = `/adls2_to_s3`
+     * See the [Microsoft docs](https://docs.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-explorer#create-a-directory) for guidance.
+
+1. Get the sample data from the link below:  
+   [customer_addresses_dim.tsv.gz](https://github.com/pivotalsoftware/pivotal-samples/raw/master/sample-data/customer_addresses_dim.tsv.gz)
+
+1. Upload the data to the directories created earlier on your S3 bucket and ADLS Gen2 container, see the links below for guidance:
```
Comment (catrionabarker) [inline docs/quickstarts/installation/s3-adlsg2_bi_lm.md:122]: Maybe
see the relevant docs for more info:

I know I'm being pedantic but you don't really 'see' a link

File: docs/quickstarts/installation/s3-adlsg2_bi_lm.md (hunk 1 above)
Comment (benradstone) [reply docs/quickstarts/installation/s3-adlsg2_bi_lm.md:13]: Yeah I prefer your suggestion here, it was a trickier one to phrase well.

File: docs/quickstarts/installation/s3-adlsg2_bi_lm.md (hunk 2 above)
Comment (benradstone) [reply docs/quickstarts/installation/s3-adlsg2_bi_lm.md:108]: I prefer this suggestion too, also adjusted the similar sentence further down to match.

**Pull request merged**